Assets/Scripts/GameSettings.cs

```diff
@@ -8,10 +8,12 @@ public enum GameState
     InGame,
     Paused,
     EndGame,
-    ItemSelect
+    ItemSelect,
+    BossVictory //For when the level 25 boss is killed
 }
 
 public static class GameSettings
 {
     public static GameState gameState = GameState.InGame;
+    public static int waveNumber = 0;
```

Comment:
would I need to update references in my scripts?

Reply:
I just added another place to track wavenumber. None of the old references should be broken. We probably should update them for consistency but it will work fine without it